pkg/apps/blackduck/components/pvc/v2/pvc.go

```diff
@@ -0,0 +1,80 @@
+package v1
+
+import (
+	horizonapi "github.com/blackducksoftware/horizon/pkg/api"
+	"github.com/blackducksoftware/horizon/pkg/components"
+	blackduckapi "github.com/blackducksoftware/synopsys-operator/pkg/api/blackduck/v1"
+	"github.com/blackducksoftware/synopsys-operator/pkg/apps/blackduck/components/utils"
+	"github.com/blackducksoftware/synopsys-operator/pkg/apps/blackduck/types"
+	"github.com/blackducksoftware/synopsys-operator/pkg/apps/store"
+	"github.com/blackducksoftware/synopsys-operator/pkg/protoform"
+	"github.com/blackducksoftware/synopsys-operator/pkg/util"
+	"k8s.io/client-go/kubernetes"
+)
+
+type BdPVC struct {
+	config     *protoform.Config
+	kubeClient *kubernetes.Clientset
+	blackduck  *blackduckapi.Blackduck
+}
+
+func NewPvc(config *protoform.Config, kubeClient *kubernetes.Clientset, blackduck *blackduckapi.Blackduck) types.PVCInterface {
+	return &BdPVC{config: config, kubeClient: kubeClient, blackduck: blackduck}
+}
+
+func (b BdPVC) GetPVCs() ([]*components.PersistentVolumeClaim, error) {
```

Comment:
This looks to be the same as in v1/pvc.go.  Maybe look at implementing in a base class an embedding in the BdPVC struct in v1 and v2?